en/02_Developer_Guides/01_Templates/06_Themes.md

```diff
@@ -27,7 +27,7 @@ composer require my_vendor/my_theme [version]
 
 [alert]
 As you've added new files to your Silverstripe CMS installation, make sure you clear the Silverstripe CMS cache by appending
-`?flush=1` to your website URL (e.g http://yoursite.com/?flush=1).
+`?flush=1` to your website URL (e.g `http://localhost/?flush=1`).
```

Comment:
```suggestion
`?flush=1` to your website URL (e.g `https://www.example.com/?flush=1`).
```

Reply:
We should probably decide globally what example domain we should use as below it's again `yoursite.com`.
`example.com` is universally understood as _the_ example domain, so I would vote for that. `yoursite.com` also works, definitely preferred over `localhost` since not many people run HTTPS on localhost, and links with HTTPS promote safer protocol practices in my view.

Reply:
Done